npm-debug.log

```diff
@@ -0,0 +1,36 @@
+0 info it worked if it ends with ok
+1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'build:js' ]
+2 info using npm@2.11.3
+3 info using node@v0.12.7
+4 verbose run-script [ 'prebuild:js', 'build:js', 'postbuild:js' ]
+5 info prebuild:js webengproject@1.0.0
+6 verbose unsafe-perm in lifecycle true
+7 info build:js webengproject@1.0.0
+8 verbose unsafe-perm in lifecycle true
+9 info webengproject@1.0.0 Failed to exec build:js script
+10 verbose stack Error: webengproject@1.0.0 build:js: `browserify src/client/js/main.js -t babelify -o dist/client/js/main.js`
+10 verbose stack Exit status 1
+10 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:213:16)
+10 verbose stack     at EventEmitter.emit (events.js:110:17)
+10 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:24:14)
+10 verbose stack     at ChildProcess.emit (events.js:110:17)
+10 verbose stack     at maybeClose (child_process.js:1015:16)
+10 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
+11 verbose pkgid webengproject@1.0.0
+12 verbose cwd /home/charles/Repos/344/WebEngProject
+13 error Linux 3.13.0-63-generic
+14 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build:js"
+15 error node v0.12.7
+16 error npm  v2.11.3
+17 error code ELIFECYCLE
+18 error webengproject@1.0.0 build:js: `browserify src/client/js/main.js -t babelify -o dist/client/js/main.js`
+18 error Exit status 1
+19 error Failed at the webengproject@1.0.0 build:js script 'browserify src/client/js/main.js -t babelify -o dist/client/js/main.js'.
+19 error This is most likely a problem with the webengproject package,
+19 error not with npm itself.
+19 error Tell the author that this fails on your system:
+19 error     browserify src/client/js/main.js -t babelify -o dist/client/js/main.js
+19 error You can get their info via:
+19 error     npm owner ls webengproject
+19 error There is likely additional logging output above.
+20 verbose exit [ 1, true ]
```

Comment:
This will get removed when the code is executing properly so I'm indifferent- siding slighting towards not ignoring- it as it's an indication during commit that you're pushing non-working code.